iceoryx_dust/include/iceoryx_dust/cxx/std_string_support.hpp

```diff
@@ -16,15 +16,13 @@
 #ifndef IOX_DUST_STD_STRING_SUPPORT_HPP
 #define IOX_DUST_STD_STRING_SUPPORT_HPP
 
-#include "iceoryx_hoofs/cxx/helplets.hpp"
-#include "iceoryx_hoofs/cxx/string.hpp"
+#include "iox/into.hpp"
+#include "iox/string.hpp"
 
 #include <string>
 
 namespace iox
 {
-namespace cxx
```

Comment:
@dkroenke That's because the `fromImpl` template is specialized here and its namespace was adapted. The 2nd level namespace in the other classes of `iceoryx_dust` will be removed on another PR before the `v3.0` release.